packages/embark-api/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "embark-api",
+  "version": "4.0.0",
```

```suggestion
  "version": "4.1.0-beta.0",
```

When introducing a package, it's initial version number should match the version number in [`lerna.json`](https://github.com/embark-framework/embark/blob/move_embark_api/lerna.json#L19) at the root of the monorepo.